IoT-Projects/ESP32/UBIDOTS_ESP32_AD8232_ECG_SYSTEM.ino/UBIDOTS_ESP32_AD8232_ECG_SYSTEM.ino

```diff
@@ -0,0 +1,104 @@
+/****************************************
+ * Include Libraries
+ ****************************************/
+#include <WiFi.h>
+#include <PubSubClient.h>
+
+#define WIFISSID "your wifi name" // Put your WifiSSID here
+#define PASSWORD "your wifi password" // Put your wifi password here
+#define TOKEN "your token" // Put your Ubidots' TOKEN
+#define MQTT_CLIENT_NAME "1234a5d6798" // MQTT client Name, please enter your own 8-12 alphanumeric character ASCII string; 
+                                           //it should be a random and unique ascii string and different from all other devices
+
+/****************************************
+ * Define Constants
+ ****************************************/
+#define VARIABLE_LABEL "sensor" // Assing the variable label
+#define DEVICE_LABEL "esp32" // Assig the device label
+
+#define SENSOR A0 // Set the A0 as SENSOR
+
+char mqttBroker[]  = "industrial.api.ubidots.com";
+char payload[100];
+char topic[150];
+// Space to store values to send
+char str_sensor[10];
+
+/****************************************
+ * Auxiliar Functions
+ ****************************************/
+WiFiClient ubidots;
+PubSubClient client(ubidots);
+
+void callback(char* topic, byte* payload, unsigned int length) {
+  char p[length + 1];
+  memcpy(p, payload, length);
+  p[length] = NULL;
+  String message(p);
```

Comment:
this is an orphan variable, it is never used in the firmware

Reply:
Actually, we are collecting the data from analog  and the esp32 is using in this project for a reason because its way better than other microcontrollers

Reply:
your answer makes no sense to the feedback, what I mean is that the variable _message_ is never used in the firmware, so it is unnecessary, just delete it.

Reply:
i just remove it,  can you please tell me the other revisions I see there is written only a thing "waiting"

Reply:
yes, instead of 'wait' you should print 'waiting'